pages/code-of-conduct.tsx

```diff
@@ -45,97 +46,130 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
         </a>
       </li>
     </StyledList>
 
-    <p>
+    <StyledPara>
       Emails and Twitter direct messages will be monitored by our media officer{' '}
-      {props.pageMetadata.conference.MediaOfficerName} as well as select subset of the organising team.
-    </p>
-
-    <p>
-      When taking a personal report, <a href={props.pageMetadata.conference.Organiser.Url}>our staff</a> will ensure you
-      are safe and cannot be overheard. They may involve other event staff to ensure your report is managed properly.
-      Once safe, we'll ask you to tell us about what happened. This can be upsetting, but we'll handle it as
-      respectfully as possible, and you can bring someone to support you. You won't be asked to confront anyone and we
-      won't tell anyone who you are.
-    </p>
-
-    <p>
+      {pageMetadata.conference.MediaOfficerName} as well as select subset of the organising team.
+    </StyledPara>
+
+    <StyledPara>
+      When taking a personal report, <a href={pageMetadata.conference.Organiser.Url}>our staff</a> will ensure you are
+      safe and cannot be overheard. They may involve other event staff to ensure your report is managed properly. Once
+      safe, we'll ask you to tell us about what happened. This can be upsetting, but we'll handle it as respectfully as
+      possible, and you can bring someone to support you. You won't be asked to confront anyone and we won't tell anyone
+      who you are.
+    </StyledPara>
+
+    <StyledPara>
       Our team will be happy to help you contact hotel/venue security, local law enforcement, local support services,
       provide escorts, or otherwise assist you to feel safe for the duration of the event. We value your attendance.
-    </p>
+    </StyledPara>
 
     <h3>Anonymous report</h3>
 
-    <p>
-      You can <a href={props.pageMetadata.conference.AnonymousReportFormUrl}>make an anonymous report here</a>. We can't
+    <StyledPara>
+      You can <a href={pageMetadata.conference.AnonymousReportFormUrl}>make an anonymous report here</a>. We can't
       follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is
       necessary to prevent a recurrence.
-    </p>
+    </StyledPara>
 
     <h3>Important contact numbers</h3>
 
     <StyledList>
       <li>
         <strong>Police:</strong>{' '}
-        <a href={props.pageMetadata.conference.ImportantContacts.Police.MapUrl}>
-          {props.pageMetadata.conference.ImportantContacts.Police.Details}
-        </a>
+        <SafeLink href={pageMetadata.conference.ImportantContacts.Police.MapUrl}>
+          {pageMetadata.conference.ImportantContacts.Police.Details}
+        </SafeLink>
+        {pageMetadata.conference.ImportantContacts.Police.Phone && (
+          <Fragment>
+            {' '}
+            <strong>Phone:</strong>{' '}
+            <a href={`tel:${pageMetadata.conference.ImportantContacts.Police.Phone}`}>
```

Comment:
Made emergency contact phone numbers links so they can be called by tapping them on your phone

Reply:
👍